### PR TITLE
Remove even more vestigial code

### PR DESCRIPTION
**Pull request by abhaybd**
Removed even more vestigial code:
- All factor graph code (it was completely undocumented and nobody knows how it works)
- Several ad-hoc tester scripts (CANTest, TestIMU, CameraTester, etc)
- Removed AR usages from sim interface. We're keeping the tag detection code around, but we're not using it since we need to revisit that later

### CODE REVIEW

File: src/filters/FilterPerfTest.cpp [hunk 1]
```diff
@@ -30,14 +30,11 @@ void usage(const char* name);
 template <typename K, typename V>
 constexpr std::unordered_map<V, K> reverseMap(const std::unordered_map<K, V>& map);
 
-enum class FilterType
-{
-	EKF,
-	PoseGraph
+enum class FilterType {
+	EKF
 };
```
Comment (QuinnP2910) [inline src/filters/FilterPerfTest.cpp:35]: An example of the weird enum linting contradiction we were talking about, GitHub CI fails

Comment (QuinnP2910) [reply src/filters/FilterPerfTest.cpp:35]: I did some research into this, I think the GitHub check just calls the clang-format-11, which defaults to formatting based on just LLVM. Adding the parameter `-style=file` should use the .clang-format in out code.

Comment (QuinnP2910) [reply src/filters/FilterPerfTest.cpp:35]: The line:
`diff=$(git clang-format-11 --diff)`
Should be
`diff=$(git clang-format-11 -style=file --diff)`
In the formatted workflow

**Review by QuinnP2910 — APPROVED** [review]
Cool, CI passes now and changes look good.

File: src/filters/FilterPerfTest.cpp (hunk 1 above)
Comment (abhaybd) [reply src/filters/FilterPerfTest.cpp:35]: Format check fixed by #295

**Pull request merged**